### ISSUE TITLE
UserWarning triggered unnecessarily; employee_fields not comprehensive 

### ISSUE THREAD

**Issue by leegenes** [smeggingsmegger/PyBambooHR#28]
looks like the UserWarning exception is raised whenever the field is not in in `employee_fields`
passing a custom field with the `fields_list` param stops the program

example:
in `request_custom_report()`

```
        get_fields = []
        field_list = [utils.underscore_to_camelcase(field) for field in field_list] if field_list else None
        if field_list:
            for f in field_list:
                if not self.employee_fields.get(f):
                    raise UserWarning("You passed in an invalid field")
                else:
                    get_fields.append(f)
```

does passing an incorrect field return an error from bamboo? 4XX?
if so, may be worth dropping this exception. thoughts?

**Comment by dant24** [comment]
@smeggingsmegger any thoughts on this?

**Comment by smeggingsmegger** [comment]
@dant24 I just set up a trial of Bamboo HR so I can take a look. I am not at the company I was working for when I originally wrote this library so I've been without a functioning BambooHR API key to use for things like this.

**Comment by smeggingsmegger** [comment]
This is now fixed in master / release 0.8.0

**Comment by dant24** [comment]
Thanks!
